pegjs/mysql.pegjs

```diff
@@ -1061,7 +1061,7 @@ create_index_definition
   = kc:(KW_INDEX / KW_KEY) __
     c:column? __
     t:index_type? __
-    de:cte_column_definition __
+    de:cte_idx_column_definition __
```

Comment:
This is require since the column expressions within an index is different than `de:cte_column_definition`, which is more permissive and allows `tableName.columnName` which is invalid in an index, for example